### ISSUE TITLE
Specification ID cannot be the same as Net Decomposition Label: invalid caseParams

### ISSUE THREAD

**Issue by yawlfoundation** [yawlfoundation/yawl#380]
```
What steps will reproduce the problem?
1. Import the attached CreateReport.yawl specification
2. Launch a CreateReport case

What is the expected output? What do you see instead?
Expected was the start of a new case. However, an exception occurs stating:
Invalid caseParams: outermost element name must match specification or root
net ID.
Tracking the problem it seems that the specification id should not be the
same as the Net Decomposition Label.

What version of the product are you using? On what operating system?
YAWL4Study 2.0, WinXP SP3

Please provide any additional information below.
Maybe this is intended behavior, but I could not find a note on it.

```

Original issue reported on code.google.com by `Jochem.V...@gmail.com` on 9 Nov 2009 at 1:06

Attachments:
* [CreateReport.yawl](https://storage.googleapis.com/google-code-attachments/yawl/issue-381/comment-0/CreateReport.yawl)
* [tomcat-log.txt](https://storage.googleapis.com/google-code-attachments/yawl/issue-381/comment-0/tomcat-log.txt)

**Comment by yawlfoundation** [comment]
```
Also, the error message in the YAWL Control Center only appears after a manual
refresh of the page (or switching to another page in control center).
```

Original comment by `Jochem.V...@gmail.com` on 9 Nov 2009 at 1:08

**Comment by yawlfoundation** [comment]
```
This is fixed in 2.0.1 (released soon)
```

Original comment by `yawl.mic...@gmail.com` on 20 Nov 2009 at 8:07
* Changed state: **Fixed**
